docs/contributing/triaging-github-issues.md

```diff
@@ -63,6 +63,18 @@ It's nice to update labels as the state of an issue changes or if the type of an
 
 Check out [the docs on issue labeling for more info](/contributing/how-to-label-an-issue/)
 
+### Questions with immediate answers
+
+- Point to existing documentation to answer the question
+- If insufficient, do the following:
+  1. Provide an answer
+  2. Label the issue with documentation
+  3. Keep it open until a PR has added the answer to the documentation
+
+If an issue comes in as a question with a known answer it can be tempting to answer it and close the issue. However, the consequence of this approach is that the answer to a question others may have is now buried in a closed issue and may be hard to surface. The preferred solution is to get that answer documented in the main Gatsby documentation.
+
+When a question comes in and it is determined to have a known answer, find the existing documentation that explains it. If that documentation does not exist, is insufficient, or unclear, it is reasonable to provide an answer in a comment in the issue. However, the issue should remain open and be labled as documentation. If you are able, you can open a PR adding the provided answer to the official documentation.
```

Comment:
This is helpful info, but it feels a little repetitive with the previous paragraph. Can they be combined somehow?
```suggestion
When a question comes in and it is determined to have a known answer, find the existing documentation that explains it. If that documentation does not exist, is insufficient, or unclear, it is reasonable to provide an answer in a comment in the issue. However, the issue should remain open and be labeled as documentation. If you are able, open a PR adding the provided answer to the official documentation.
```

Reply:
Ya, originally this was a paragraph explanation of the ordered list that followed. We moved the list up top. If we feel that adequately covers it I'm fine to remove it.